### ISSUE TITLE
Changing theme mess up extraterm UI

### ISSUE THREAD

**Issue by Devligue** [sedwards2009/extraterm#83]
I expearienced it first when I initially installed extraterm on Windows and played a bit with its settings. I don't really remember how I fixed it (probably by reinstalling), but I didn't touch the theme settings since then. And today it happened on my Linux (Manjaro).

Basically when I changed the theme from Atom Dark to Atom Light (I can't remember the full names, and you will see I can't check it anymore), the extraterm went blank white and tabs shifted to bottom. It seems that everything is still working fine except that the whole interface is shifted to the bottom and out of the screen so I see only the tabs but not the content. And I cant fix it by changing the theme back because settings also open in the tab which is out of window view. 

tl;dr Here it is what happened:
![extraterm](https://user-images.githubusercontent.com/12434847/38055978-6ee61a20-32db-11e8-8a42-6df837976894.png)

UPDATE:
Reinstalling does not help obviously, because the settings are saved in `~/.config/extraterm`. Just delete this directory if anyone faces similar problem and everything goes back to normal. But this does not make the bug less buggy ^^

**Comment by sedwards2009** [comment]
Thanks for reporting. I can't say I've ever seen this before. You say that it was triggered by playing around with the theme settings? and using the native window controls too?

**Comment by Devligue** [comment]
Just:

Open Extraterm > Settings > Theme > Interface Theme
then change from `Atomic Dark UI Theme` (which is default after installation) to `Atomic Light UI Theme`.

I can reproduce it every time both on Manjaro and on Windows 10. Feel free to ask me for some kind of logs if there are any, or if I can make them with sort of --debug option or sth.

If that matters, on manjaro I installed extraterm from AUR. On Windows I don't remember, sorry.

**Comment by Devligue** [comment]
I managed to narrow it down a bit.

It seems that it does not happen right away. After deleting ~/.config/extraterm directory (or fresh installation for that matter) all themes work fine, and I can switch between them. If I return back to default theme settings before closing it will also be fine next time after opening extraterm. The trigger condition is closing extraterm with one of Theme settings changed (worked both for `title bar` setting changed and/or `interface theme` changed). As it turns out it doesn't matter to which theme I switch. After opening extraterm again UI is messed up.

For example here is the effect of some other randomly selected theme:
![extraterm2](https://user-images.githubusercontent.com/12434847/38059496-f0d187da-32e6-11e8-8791-97fb8ca7ba21.png)

**Comment by sedwards2009** [comment]
It looks like the CSS generation failed. I'll look into it tonight.
